lib/src/DatePicker/DatePickerToolbar.tsx

```diff
@@ -1,23 +1,28 @@
 import * as React from 'react';
-import clsx from 'clsx';
-import ToolbarButton from '../_shared/ToolbarButton';
 import PickerToolbar from '../_shared/PickerToolbar';
+import { DatePickerView } from './DatePicker';
+import { PenIcon } from '../_shared/icons/PenIcon';
 import { useUtils } from '../_shared/hooks/useUtils';
-import { makeStyles } from '@material-ui/core/styles';
 import { ToolbarComponentProps } from '../Picker/Picker';
 import { isYearAndMonthViews, isYearOnlyView } from '../_helpers/date-utils';
+import { Typography, makeStyles, IconButton, Grid, Theme } from '@material-ui/core';
 
-export const useStyles = makeStyles(
+export const useStyles = makeStyles<Theme, { isLandscape: boolean }>(
   {
     toolbar: {
+      color: 'white',
       flexDirection: 'column',
       alignItems: 'flex-start',
+      justifyContent: 'space-between',
+      paddingTop: 16,
+      paddingBottom: 16,
+      padding: props => (props.isLandscape ? 16 : undefined),
```

Comment:
I would avoid using style function in any of the components, it's not ready for prime-time.

Reply:
:( I was happy that it will be possible to get rid of clsx 😢. Do you know when it will be ok? Because docs are saying that everything is stable enough.

Reply:
I wish we were able to use them in the core components too. We don't as we are speaking. I think that it would either require the styled-components migration or to use react-jss internally. Performance is OK but not great. It's fine for the Box, but I wouldn't use them as a systemic solution.